mlflow/utils/rest_utils.py

```diff
@@ -270,12 +270,16 @@ def extract_all_api_info_for_service(service, path_prefix):
     return res
 
 
+def get_single_trace_endpoint(request_id):
+    return f"{_REST_API_PATH_PREFIX}/mlflow/traces/{request_id}"
```

Comment:
Maybe we can just make `{_REST_API_PATH_PREFIX}/mlflow/traces` a constant?

Reply:
+1